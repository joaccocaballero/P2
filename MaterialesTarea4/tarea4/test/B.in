crearBalanceado 0
crearBalanceado 1 (50, 0)
crearBalanceado 7 (1, 0) (2, 0) (3, 0) (4, 0) (5, 0) (6, 0) (7, 0) 
crearBalanceado 8 (10, 1) (20, 1) (30, 1) (40, 1) (50, 1) (60, 1) (70, 1) (80, 1) 
unionAbbs 0 0
unionAbbs 6 (40,0) (20,0) (10,0) (50,0) (30,0) (60,0) 7 (5,0) (25,0) (55,0) (45,0) (35,0) (15,0) (65,0)
Fin
