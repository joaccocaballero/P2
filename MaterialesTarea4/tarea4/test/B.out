1>
2>
(50,0.00)
3>
--(7,0.00)
-(6,0.00)
--(5,0.00)
(4,0.00)
--(3,0.00)
-(2,0.00)
--(1,0.00)
4>
---(80,1.00)
--(70,1.00)
-(60,1.00)
--(50,1.00)
(40,1.00)
--(30,1.00)
-(20,1.00)
--(10,1.00)
5>
6>
---(65,0.00)
--(60,0.00)
---(55,0.00)
-(50,0.00)
---(45,0.00)
--(40,0.00)
(35,0.00)
---(30,0.00)
--(25,0.00)
---(20,0.00)
-(15,0.00)
---(10,0.00)
--(5,0.00)
7>Fin.
